android/assets/jsons/Civ V - Vanilla/Units.json

```diff
@@ -1532,6 +1532,21 @@
 			"[-1] Visibility Range", "Unbuildable", "Religious Unit", "Hidden when religion is disabled"],
 		"movement": 4,
 		"religiousStrength": 1000
+	},
+	{
+		"name": "Inquisitor",
+		"unitType": "Civilian",
+		"uniques": ["Prevents spreading of religion to the city it is next to",
+			"May remove heresy [1] times",
+			// ToDo: change this to an _enhanced_ religion when that is implemented
+			"Can be purchased with [Faith] [in all cities in which the majority religion is a major religion]",
+			"[+1] Visibility Range", "Hidden when religion is disabled", "Unbuildable", "Religious Unit"
+		],
+		"movement": 3,
+		"religiousStrength": 1000,
+		"civilopediaText": [
+			{"text": "Can remove foreign religions from your own cities."}
```

Comment:
If you need text to explain what the unique enables, it's probably not user friendly enough.